src/node/routes/index.ts

```diff
@@ -117,4 +142,11 @@ export const register = async (app: Express, server: http.Server, args: Defaulte
   }
 
   app.use(errorHandler)
+
+  const wsErrorHandler: express.ErrorRequestHandler = async (err, req) => {
+    logger.error(`${err.message} ${err.stack}`)
+    ;(req as WebsocketRequest).ws.destroy(err)
```

Comment:
I believe we want to use `end` here to ensure at least everything written to the connection so far is flushed correctly.

Reply:
That makes sense, I'll change this and the other one to `end`.